public/app/plugins/panel/timeseries/models.cue

```diff
@@ -23,18 +23,12 @@ Family: {
         [
             {
                 PanelOptions: {
+                    // FIXME idk where this is coming from but various devenv dashes have it
+                    graph?: {...}
```

Comment:
looking though the dashboards, I think we can just search/replace:
```
"options": {
        "graph": {},
        "legend": {
```
with
```
"options": {
        "legend": {
```

if there are still issues, then lets figure that out, but looks like everything is empty object

Reply:
i should've been clearer in the comment - it's not just that devenv dashboards have it. They also still recreate it when re-exported from the frontend. So removing these now would create another bit of nasty one-off maintenance.

IMO better to change it in a follow-up when we fix the underlying behavior that's causing the field to still be generated.

Reply:
Hymm -- I just tried removeing all `graph` options -- including:
![image](https://user-images.githubusercontent.com/705951/129666632-49e57a29-469e-44d3-8cab-f8c069cdb239.png)

and I don't see it getting re-generated.  The string "realTimeUpdates" does not exist in our code :thinking: 

But I am totally on board merging this and then adding another issue to clean up more.